src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -461,6 +461,8 @@ data class IndividualUser(
 
   override fun canReadProjectScores(projectId: ProjectId) = isReadOnlyOrHigher()
 
+  override fun canReadProjectVariableOwners(projectId: ProjectId) = isReadOnlyOrHigher()
```

Comment:
Is `projectId` needed here?

Reply:
Not needed per se, more following the pattern of making permissions fine-grained at the level of this API so we can easily make the logic more complex without having to change all the call sites (e.g., maybe we'll eventually have a team that only deals with African projects and doesn't have permission to see Asian ones).